benefits/in_person/views.py

```diff
@@ -147,6 +166,7 @@ def reenrollment_error(request):
 
 def retry(request):
     """View handler for card verification failure."""
+    enrollment_analytics.returned_retry(request, enrollment_method=models.EnrollmentMethods.IN_PERSON)
```

Comment:
This one can't be moved since the user is sent here from JavaScript.

Would _could_ enforce that this route is `POST`-only and make the from `POST` back to here... that doesn't have to happen in this PR though.

Reply:
Right, I'll make a ticket for it 👍